src/pages/pool/create.vue

```diff
@@ -80,6 +96,21 @@ onMounted(async () => {
 const doSimilarPoolsExist = computed(() => similarPools.value.length > 0);
 const validTokens = computed(() => tokensList.value.filter(t => t !== ''));
 
+const unknownTokens = computed(() => {
+  if (dynamicDataLoading.value) return [];
+  return validTokens.value.filter(token => {
+    return priceFor(token) === 0;
```

Comment:
This is a difficult one since the value returned from `priceFor` will be 0 if its not there, I'm not sure if we should adjust the existing behaviour and nullify it, if there is no value?
@garethfuller thoughts?